packages/libssh-0001-install-modified-pc-file.patch

```diff
@@ -15,7 +15,7 @@ index f288b94..5728e43 100644
  Description: The SSH Library
  Version: @PROJECT_VERSION@
  Libs: -L${libdir} -lssh
-+Libs.private: -lmbedtls -lmbedx509 -lmbedcrypto -lz -lwsock32 -lbcrypt -lws2_32 -lpthread
++Libs.private: -lcrypto -lssl -lz -lbrotlicommon -lbrotlidec -lbrotlienc -lwsock32 -lcrypt32 -lws2_32 -lpthread
```

Comment:
```suggestion
+Libs.private: -lcrypto -lssl -lz -lwsock32 -lcrypt32 -lws2_32 -lpthread
+Requires.private: libbrotlidec libbrotlienc
```
Otherwise, ffmpeg cannot find brotli's lib when configure libssh on gcc